mlflow/R/mlflow/R/model-registry.R

```diff
@@ -26,6 +26,82 @@ mlflow_create_registered_model <- function(name, tags = NULL, description = NULL
   return(response$registered_model)
 }
 
+#' Get a registered model
+#'
+#' Retrieves a registered model from the Model Registry.
+#'
+#' @param name The name of the model to retrieve.
+#' @template roxlate-client
+#' @export
+mlflow_get_registered_model <- function(name, client = NULL) {
+  client <- resolve_client(client)
+
+  response <- mlflow_rest(
+    "registered-models",
+    "get",
+    client = client,
+    verb = "GET",
+    version = "2.0",
+    query = list(
```

Comment:
should we fit `query = list(name = name)` on one line?

Reply:
Done.